LibNFS.xs

```diff
@@ -983,6 +983,33 @@ static void _set_unix_authn(pTHX_ struct nfs_context* nfs, SV* value_sv) {
     nfs_set_auth(nfs, auth);
 }
 
+#ifdef NLNFS_NFS_SET_READDIR_MAX_BUFFER_SIZE
+static void _set_readdir_max_buffer_size(pTHX_ struct nfs_context* nfs, SV* value_sv) {
+
+    if (SvROK(value_sv) && (SvTYPE(SvRV(value_sv)) == SVt_PVAV)) {
+        AV* av = (AV*) SvRV(value_sv);
+        uint32_t nums_count = 1 + av_len(av);
+        if (nums_count != 2) croak("“%s” must contain exactly 2 numbers", "readdir_buffer");
+
+        uint32_t nums[nums_count];
+
+        for (unsigned n=0; n<nums_count; n++) {
+            SV** svp = av_fetch(av, n, 0);
+            assert(svp);
```

Comment:
NB: Perl black-holes `assert()`. It _should_ never fail, though, since you just verified the array’s length above.

Reply:
I just copied this from _set_unix_authn().

Reply:
Yeah it’s fine; just FYI.